script.js

```diff
@@ -11,3 +11,273 @@ function openDropdownItem(menuElement, menuLink) {
+                <div class='work-description-section'>
+                    <h2 class='work-title'>${currentProject.name}</h2>
+                    <div class="work-pointers">
+                      <h3>${currentProject.pointersTitle}</h3>
+                      <div class="work-pointers-bullets"></div>
+                      <p>${currentProject.pointersLevel}</p>
+                      <div class="work-pointers-bullets"></div>
+                      <p>${currentProject.pointersYear}</p>
+                    </div>
+                    <p class='work-description'>${currentProject.description}</p>
+                    <ul class='tags'>
+                        ${tech}
+                    </ul>
+                    <a href='#!' class='project-link' onclick='openDesktopPopUp(${i})' >
+                     See Project
+                    </a>
+                </div>
+            </div>
+        `;
+  }
+  return data;
+}
+document.querySelector('#desktop-works').innerHTML = generateDesktopProjectData(projectData);
+function openDesktopPopUp(counter) {
+  const currentProjectData = desktopProjectData[counter];
+  const desktopTechnologies = getTechnologies(
+    desktopProjectData[counter].technologies,
+  );
+  const data = `
+    <div class='desktop-popup' >
+        <div class='desktop-popup-title'>
+            <h2 class='work-title'>${currentProjectData.name}</h2>
+        <i class='fa-regular fa-x desktop-dropdown-x-icon' onclick='closeDesktopPopUp()'></i>
+        </div>
+        <div class="work-pointers">
+          <h3>${currentProjectData.pointersTitle}</h3>
+          <div class="work-pointers-bullets"></div>
+          <p>${currentProjectData.pointersLevel}</p>
+          <div class="work-pointers-bullets"></div>
+          <p>${currentProjectData.pointersYear}</p>
+      </div>
+      <img src='${currentProjectData.featuredImage}' class='desktop-popup-work-image' alt='image of a website hero-section'>
+        <div class='desktop-popup-description'>
+            <div class='desktop-popup-work-description'>
+                <p>${currentProjectData.description}</p>
+            </div>
+            <div class='desktop-popup-items'>
+                <ul class=' desktop-popup-tags'>
+                ${desktopTechnologies}
+                </ul>
+                <a href='${currentProjectData.linkToLiveVersion}' class=' desktop-popup-project-link'>
+                See live
+                    <img src='./images/Popup-icon.png' alt='image of a small icon' class='popup-icon'>
+                </a>
+                <a href='${currentProjectData.linkToSource}' class=' desktop-popup-project-link'>
+                See Source
+                    <img src='./images/github-vector.png' alt='github icon' class='popup-icon'>
+                </a>
+            </div>
+        </div>
+    </div>
+    `;
+  desktopPopUp.innerHTML = data;
+  desktopPopUp.classList.remove('hide-desktop-popup');
+}
```

Comment:
- I see that you have two different functions for displaying your pop-up for both desktop and mobile versions, this is not considered the best practice. I suggest that you have just one function to display your pop-up modal, and just one array of projects to store your project. This will help make your project cleaner and easier to maintain. Please fix this 🙏